core/src/main/java/org/infinispan/factories/SizeCommandHandlerFactory.java

```diff
@@ -0,0 +1,12 @@
+package org.infinispan.factories;
+
+import org.infinispan.factories.annotations.DefaultFactoryFor;
+import org.infinispan.reactive.size.FallbackSizeCommandHandler;
+
+@DefaultFactoryFor(classes = FallbackSizeCommandHandler.class)
+public class SizeCommandHandlerFactory extends AbstractNamedCacheComponentFactory implements AutoInstantiableFactory {
+
+   @Override public Object construct(String name) {
+      return new FallbackSizeCommandHandler();
```

Comment:
This seems unneded. Or was the point to add another impl based on the cache config?

Reply:
ended up this was unnecessary